lib/limiter/mixin.rb

```diff
@@ -6,9 +6,16 @@ def limit_method(method, rate:, interval: 60)
       queue = RateQueue.new(rate, interval: interval)
 
       mixin = Module.new do
-        define_method(method) do |*args|
-          queue.shift
-          super(*args)
+        if RUBY_VERSION < "2.7"
```

Comment:
An alternative solution to checking the Ruby version is to release v2 of this gem and support only Ruby 2.7+.